kernel/packages/unity-interface/BrowserInterface.ts

```diff
@@ -286,6 +292,15 @@ export class BrowserInterface {
     }
   }
 
+  public SendAuthentication(data: { rendererAuthenticationType: RendererAuthenticationType }) {
+    let providerType: ProviderType | null = null
+    if (data.rendererAuthenticationType === RendererAuthenticationType.WALLET_CONNECT) {
+      providerType = ProviderType.WALLET_CONNECT
+    }
+
+    authenticateWhenItsReady(providerType)
+  }
```

Comment:
why not sending a string as payload instead of trying to match enums of integers and maintaining one enum on kernel and other on the renderer? it would be probably easier to extend in the future
`ProviderType.WALLET_CONNECT -> "wallet_connect"`